src/main/java/com/kilopo/kosshop/service/Impl/ProducerServiceImpl.java

```diff
@@ -0,0 +1,22 @@
+package com.kilopo.kosshop.service.Impl;
+
+import com.kilopo.kosshop.DAO.ProducerDAO;
+import com.kilopo.kosshop.entity.Producer;
+import com.kilopo.kosshop.service.ProducerService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public class ProducerServiceImpl implements ProducerService {
+    @Autowired
+    ProducerDAO producerDAO;
+
+    public boolean addProducer(Producer producer) {
+        try {
```

Comment:
i think better way to catch exception in dao method and remove it from all services